packages/web/src/scss/components/FieldGroup/_FieldGroup.scss

```diff
@@ -21,7 +19,7 @@ $_field-name: 'FieldGroup';
 
 .FieldGroup__fields {
     display: grid;
```

Comment:
Question is if we are OK with the consequences because this expands all children over the `FieldGroup` width so `Checkbox`es and `Radio`s are clickable even far outside of their labels…

How about justifying grid children to start?

![image](https://github.com/user-attachments/assets/ebc9ab07-4d31-45f3-9faa-6e5105c343c5)